index.html

```diff
@@ -38,5 +38,161 @@ <h2>
+                </ul>
+                <button>See Project</button>
+            </div>
+            <div class="card-works">
+              <h1>Profesional Art Printing Data</h1>
+              <p>
+                  A daily selection of privately personalized reads; 
+                  no accounts or sign-ups required. 
+                  has been the industry's standard
+                </p>
+                <ul>
+                  <li class="html">html</li>
+                  <li class="boostrap">boostrap</li>
+                  <li class="ruby">ruby</li>
+                </ul>
+                <button>See Project</button>
+            </div>
+            <div class="card-works">
+              <h1>Profesional Art Printing Data</h1>
+              <p>
+                  A daily selection of privately personalized reads; 
+                  no accounts or sign-ups required. 
+                  has been the industry's standard
+                </p>
+                <ul>
+                  <li class="html">html</li>
+                  <li class="boostrap">boostrap</li>
+                  <li class="ruby">ruby</li>
+                </ul>
+                <button>See Project</button>
+            </div>
+            <div class="card-works">
+              <h1>Profesional Art Printing Data</h1>
+              <p>
+                  A daily selection of privately personalized reads; 
+                  no accounts or sign-ups required. 
+                  has been the industry's standard
+                </p>
+                <ul>
+                  <li class="html">html</li>
+                  <li class="boostrap">boostrap</li>
+                  <li class="ruby">ruby</li>
+                </ul>
+                <button>See Project</button>
+            </div>
+            <div class="card-works">
+              <h1>Profesional Art Printing Data</h1>
+              <p>
+                  A daily selection of privately personalized reads; 
+                  no accounts or sign-ups required. 
+                  has been the industry's standard
+                </p>
+                <ul>
+                  <li class="html">html</li>
+                  <li class="boostrap">boostrap</li>
+                  <li class="ruby">ruby</li>
+                </ul>
+                <button>See Project</button>
+            </div>
+            <div class="card-works">
+              <h1>Profesional Art Printing Data</h1>
+              <p>
+                  A daily selection of privately personalized reads; 
+                  no accounts or sign-ups required. 
+                  has been the industry's standard
+                </p>
+                <ul>
+                  <li class="html">html</li>
+                  <li class="boostrap">boostrap</li>
+                  <li class="ruby">ruby</li>
+                </ul>
+                <button>See Project</button>
+            </div>
+            <div class="about-me">
```

Comment:
- [ ] As mentioned in the previous review, Kindly consider adding a background image to your `About` section.
Your design :point_down: 
![image](https://user-images.githubusercontent.com/20554583/199916670-00f59240-9106-4c25-b05c-0ebc0dcacd10.png)

Template design :point_down: 
![image](https://user-images.githubusercontent.com/20554583/199916693-c4223b8c-6d23-4f43-a7cb-24e016e91896.png)

Reply:
please kindly check it well it has been fix